catalog/task/apply-mapping/0.3/README.md

```diff
@@ -39,6 +39,8 @@ tasks:
 ## Changes since 0.2
 
   * Base image was changed from `release-utils` to `release-base-image`
+  * The syntax for `taskRef.bundle` and `pipelineRef.bundle` is deprecated,
```

Comment:
```suggestion
  * The syntax for `taskRef.bundle` and `pipelineRef.bundle` are deprecated,
```

Reply:
I think `is` is correct here. The thing being talked about is "the syntax." Just like if we shared a desk that had broken, you'd say "the desk for Leandro and Johnny is broken" not "the desk for Leandro and Johnny are broken"

Reply:
I see! I read that if you have two nouns, the it would be _are_. But yes, they way you added it makes more sense. Thanks!